server/server.js

```diff
@@ -4,12 +4,14 @@ const express = require('express')
 const server = express()
 
 const starterRoute = require('./routes/starter')
+const users = require('./routes/usersRoutes')
 const cohortRoutes = require('./routes/cohortRoutes')
 
 server.use(express.json())
 server.use(express.static(path.join(__dirname, './public')))
 
 server.use('/', starterRoute)
+server.use('/api/v1/users', users)
```

Comment:
Could you please change this line to `server.use('/api/v1/users', usersRoutes)`
Thanks!

Reply:
Thanks @emilyparkes this is now compete